### PR TITLE
Add annotations and network events as labels in Stackdriver v1 API.

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

File: exporters/trace_stackdriver/src/main/java/io/opencensus/exporter/trace/stackdriver/StackdriverV1ExporterHandler.java [hunk 1]
```diff
@@ -132,6 +155,70 @@ private static String encodeTraceId(TraceId traceId) {
...
+      if (first) {
+        first = false;
+        stringBuilder.append(entry.getKey());
+        stringBuilder.append("=");
+        stringBuilder.append(attributeValueToString(entry.getValue()));
+      } else {
+        stringBuilder.append(", ");
+        stringBuilder.append(entry.getKey());
+        stringBuilder.append("=");
+        stringBuilder.append(attributeValueToString(entry.getValue()));
```
Comment (sebright) [inline exporters/trace_stackdriver/src/main/java/io/opencensus/exporter/trace/stackdriver/StackdriverV1ExporterHandler.java]: Suggestion: These three lines could be moved out of the if and else blocks:

        stringBuilder.append(entry.getKey());
        stringBuilder.append("=");
        stringBuilder.append(attributeValueToString(entry.getValue()));

Comment (bogdandrutu) [reply exporters/trace_stackdriver/src/main/java/io/opencensus/exporter/trace/stackdriver/StackdriverV1ExporterHandler.java]: Done.

File: exporters/trace_stackdriver/src/main/java/io/opencensus/exporter/trace/stackdriver/StackdriverV1ExporterHandler.java [hunk 2]
```diff
@@ -95,6 +102,22 @@ private Trace generateTrace(SpanData spanData) {
       spanBuilder.putLabels(label.getKey(), attributeValueToString(label.getValue()));
     }
 
+    // Add Annotations as labels in the v1 API
+    int seq = 0;
+    for (TimedEvent<Annotation> annotations : spanData.getAnnotations().getEvents()) {
```
[the remark this answers is not in the document]
Comment (bogdandrutu) [reply exporters/trace_stackdriver/src/main/java/io/opencensus/exporter/trace/stackdriver/StackdriverV1ExporterHandler.java]: Done.

File: exporters/trace_stackdriver/src/main/java/io/opencensus/exporter/trace/stackdriver/StackdriverV1ExporterHandler.java [hunk 3]
```diff
@@ -132,6 +155,67 @@ private static String encodeTraceId(TraceId traceId) {
...
+    boolean first = true;
+    for (Map.Entry<String, AttributeValue> entry : attributes.entrySet()) {
+      if (first) {
+        first = false;
+      } else {
+        stringBuilder.append(", ");
+      }
+      stringBuilder.append(entry.getKey());
+      stringBuilder.append("=");
+      stringBuilder.append(attributeValueToString(entry.getValue()));
```
Comment (adriancole) [inline exporters/trace_stackdriver/src/main/java/io/opencensus/exporter/trace/stackdriver/StackdriverV1ExporterHandler.java]: Do these values have a chance of needing json escaping?

Comment (bogdandrutu) [reply exporters/trace_stackdriver/src/main/java/io/opencensus/exporter/trace/stackdriver/StackdriverV1ExporterHandler.java]: Using proto + gRPC so no need to json escape them 👍

**Comment by bogdandrutu** [comment]
My code is bug free 🥇 no need for tests.

We are very soon switching to v2 api, and this exporter is full of hacks to be able to use v1. I will add tests (I promise if this survives more than 1 month).

**Pull request merged**